usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/PetData.java

```diff
@@ -110,6 +135,106 @@ public void setRecommendedKcal(Double recommendedKcal) {
         this.recommendedKcal = recommendedKcal;
     }
 
+    /**
+     * Checks that field has the correct format for a Pet simple attribute.
+     * @param field Name of the attribute.
+     */
+    public static void checkSimpleField(String field) {
+        if (!GENDER.equals(field) && !BREED.equals(field) && !BIRTH.equals(field)
+            && !PATHOLOGIES.equals(field) && !NEEDS.equals(field) && !RECOMMENDED_KCAL.equals(field)) {
+            throw new IllegalArgumentException("Field does not exists");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "Field does not exists" appears 3 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5479466)